WordPress/src/main/java/org/wordpress/android/ui/posts/PostListItemViewHolder.kt

```diff
@@ -118,6 +120,9 @@ sealed class PostListItemViewHolder(
         showFeaturedImage(data.imageUrl)
         updateProgressBarState(data.progressBarState)
         uiHelpers.updateVisibility(disabledOverlay, data.showOverlay)
+        if (data.disableRippleEffect) {
```

Comment:
Having an `if` statement without `else` statement in ViewHolders is a common source of bugs since the items can be recycled. I realize it's not an issue in this case as all the items either are or aren't in "Trashed" tab. However, I'd still consider adding an else statement in case the behavior changes in the future. Wdyt?

Reply:
Good point, updated in f2eb152